pkg/webhook/webhook.go

```diff
@@ -398,6 +406,38 @@ func (w *Config) buildFleetValidatingWebhooks() []admv1.ValidatingWebhook {
 			},
 			TimeoutSeconds: longWebhookTimeout,
 		},
+		{
+			Name:                    "fleet.managedresource.validating",
+			ClientConfig:            w.createClientConfig(managedresource.ValidationPath),
+			FailurePolicy:           &failFailurePolicy,
+			SideEffects:             &sideEffortsNone,
+			AdmissionReviewVersions: admissionReviewVersions,
+			Rules: []admv1.RuleWithOperations{
+				{
+					Operations: []admv1.OperationType{
+						admv1.Create, admv1.Update, admv1.Delete,
+					},
+					Rule: createRule(
```

Comment:
I think this is interpreted as the cross product, which is not quite right.  I think you need to call createRule() three times, once for each GV.

Reply:
Split them into 4, one for each resource type.  corev1 namespace can then have a different scope than corev1 resourcequota.